src/store/sdp/index.js

```diff
@@ -5,12 +5,12 @@ import { firebase, getDatabase, ref, get, child } from '../../firebase';
 const state = {
     spotDetails: null,
     ownerInfoDetails: null,
-    selectedSpot: [],
     isAvailable: false,
     loading: false,
     title: '',
     thumbnail: [],
     images: [],
+    userLatLng: { lat: 37.77392, lng: -122.223 },
```

Comment:
why this defaulf lat lng?

Reply:
set to banglore, it is default value if user deny to share the locations